infra/logging/src/main/java/com/simprints/infra/logging/writers/OptionalTagFormatter.kt

```diff
@@ -0,0 +1,19 @@
+package com.simprints.infra.logging.writers
+
+import co.touchlab.kermit.Message
+import co.touchlab.kermit.MessageStringFormatter
+import co.touchlab.kermit.Severity
+import co.touchlab.kermit.Tag
+
+internal class OptionalTagFormatter : MessageStringFormatter {
+    override fun formatTag(tag: Tag): String = "(${tag.tag})"
+
+    override fun formatMessage(
+        severity: Severity?,
+        tag: Tag?,
+        message: Message,
+    ): String {
+        if (tag?.tag.isNullOrBlank()) return message.message
+        return "${formatTag(tag)} ${message.message}"
```

Comment:
Previously it was `[TAG] Message` which IMO is cleaner

Reply:
This is the same, just with a check that tag is present. I will make the brackets square, to keep it closer to the old approach.